### PR TITLE
Added CollectionView ItemSwipe

### PR DESCRIPTION
**Pull request by kerpecheva**

### CODE REVIEW

File: controls/collectionview/item-swipe/overview.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+.NET MAUI CollectionView provides item swipe feature - when users swipe, they reveal a designated custom view with buttons, images etc.
+
+The image below shows how swiping can reveal buttons on the left and right:
+
+![.NET MAUI CollectionView Item Swipe](../images/collectionview-item-swipe.png)
+
+## Properties
+
+You can use the following `RadCollectionView` properties to configure the Item Swipe feature:
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/overview.md:20]:
```suggestion
You can use the following `RadCollectionView` properties to configure the item swipe feature:
```

File: controls/collectionview/item-swipe/overview.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+- `IsItemSwipeEnabled`(`bool`)&mdash;Indicates whether the items of the CollectionView can be swiped. The default value is `False`.
+- `SwipeThreshold`(`double`)&mdash;Defines the length (in pixels) of the swipe gesture that is required to trigger the feature.
+- `SwipeOffset`(`Thickness`)&mdash;Defines the distance that a swiped item will be moved to and stay there until the swipe gets closed.
+- `StartSwipeTemplate`(`DataTemplate`)&mdash;Defines the content that is visualized when swipe is performed to left in vertical orientation or up in horizontal orientation.
+- `EndSwipeTemplate`(`DataTemplate`)&mdash;Defines the content that is visualized when swipe is performed to right in vertical orientation or down in horizontal orientation.
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/overview.md:26]:
```suggestion
- `EndSwipeTemplate`(`DataTemplate`)&mdash;Defines the content that is visualized for a swipe-right (in vertical orientations) or a swipe-down (in horizontal orientations).
```

File: controls/collectionview/item-swipe/item-swipe-events.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+position: 1
+slug: collectionview-item-swipe-events
+tags: item-swipe, collectionview, swiping, events
+---
+
+## .NET MAUI CollectionView Swiping Events
+
+The following `RadCollectionView` events are related to the item swiping feature:
+
+- `SwipeStarting`&mdash;Occurs when item is about to be swiped. The event arguments are of the `CollectionViewSwipeStartingEventArgs` type that provides the following properties:
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/item-swipe-events.md:14]:
```suggestion
- `SwipeStarting`&mdash;Occurs when an item is about to be swiped. The event arguments are of the `CollectionViewSwipeStartingEventArgs` type that provides the following properties:
```

File: controls/collectionview/item-swipe/item-swipe-events.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  - `Item`(`object`)&mdash;The item that will be swiped.
+  - `Cancel`(`bool`)&mdash;If you set this value to `false`, the swiping will be canceled.
+- `Swiping`&mdash;Occurs while the user is swiping the item. The event arguments are of the `CollectionViewSwipingEventArgs` type that provides the following properties:
+  - `Item`(`object`)&mdash;The item that is being swiped.
+  - `Offset`(`double`)&mdash;The offset of the swiped item from its initial position.
+- `SwipeCompleted`&mdash;Occurs when swiping of an item is completed. The event arguments are of the `CollectionViewSwipeCompletedEventArgs` type that provides the following properties:
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/item-swipe-events.md:20]:
```suggestion
- `SwipeCompleted`&mdash;Occurs when the swiping of an item is completed. The event arguments are of the `CollectionViewSwipeCompletedEventArgs` type that provides the following properties:
```

File: controls/collectionview/item-swipe/item-swipe-events.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  - `Item`(`object`)&mdash;The item that has been swiped.
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/item-swipe-events.md:21]:
```suggestion
  - `Item`(`object`)&mdash;The item that was swiped.
```

File: controls/collectionview/item-swipe/item-swipe-commands.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+page_title: .NET MAUI CollectionView Documentation - Item Swipe Commands
+description: Review the Telerik UI for .NET MAUI CollectionView Item Swipe Commands.
+position: 2
+slug: collectionview-item-swipe-commands
+tags: item-swipe, collectionview, swiping, commands
+---
+
+## .NET MAUI CollectionView Swiping Commands
+
+The .NET MAUI CollectionView provides the  following commands related to swipe actions:
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/item-swipe-commands.md:12]:
```suggestion
The .NET MAUI CollectionView provides the following commands related to swipe actions:
```

File: controls/collectionview/item-swipe/item-swipe-commands.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+- `SwipeStartingCommand`(`ICommand`)&mdash;Occurs when item is about to be swiped. The command parameter is of the `CollectionViewSwipeStartingCommandContext` type that provides the following properties:
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/item-swipe-commands.md:14]:
```suggestion
- `SwipeStartingCommand`(`ICommand`)&mdash;Occurs when an item is about to be swiped. The command parameter is of the `CollectionViewSwipeStartingCommandContext` type that provides the following properties:
```

File: controls/collectionview/item-swipe/item-swipe-commands.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+  - `Item`(`object`)&mdash;The item that will be swiped.
+  - `Cancel`(`bool`)&mdash;If you set this value to `false`, the swiping will be canceled.
+- `Swiping`(`ICommand`)&mdash;Occurs while the user is swiping the item. The command parameter is of the `CollectionViewSwipingCommandContext` type that provides the following properties:
+  - `Item`(`object`)&mdash;The item that is being swiped.
+  - `Offset`(`double`)&mdash;The offset of the swiped item from its initial position.
+- `ItemSwipeCompleted`(`ICommand`)&mdash;Occurs when swiping of an item is completed. TThe command parameter is of the `CollectionViewSwipeCompletedCommandContext` type that provides the following properties:
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/item-swipe-commands.md:20]:
```suggestion
- `ItemSwipeCompleted`(`ICommand`)&mdash;Occurs when the swiping of an item is completed. The command parameter is of the `CollectionViewSwipeCompletedCommandContext` type that provides the following properties:
```

File: controls/collectionview/item-swipe/item-swipe-commands.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  - `Item`(`object`)&mdash;The item that has been swiped.
```
Comment (ikoevska) [inline controls/collectionview/item-swipe/item-swipe-commands.md:21]:
```suggestion
  - `Item`(`object`)&mdash;The item that was swiped.
```

**Review by ikoevska — CHANGES_REQUESTED** [review]
Same issue as with drag-and-drop. You need to fix the config.yml to render the section name properly.

image

**Comment by kerpecheva** [comment]
Updated section title in config

**Review by ikoevska — APPROVED** [review]

**Pull request merged**